### PR TITLE
YogitaP_Coding_Exam_4_45min

### PR DESCRIPTION
**Pull request by YogitaPOct2020**
image

image

### CODE REVIEW

File: src/yogita/FindIdenticalElement.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	void identicalElementFRomArray(int[] array1, int[] array2) {
+		System.out.println("Array1 is:" + Arrays.toString(array1));
+		System.out.println("Array2 is:" + Arrays.toString(array2));
+
+		System.out.println("Identical Elemets From Array Are:");
+		for (int i = 0; i < array1.length; i++) {
+			boolean flag = false;
+			for (int j = 0; j < array2.length; j++) {
+				if (array1[i] == array2[j])
+					flag = true;
```
Comment (harshvegada) [inline src/yogita/FindIdenticalElement.java:27]: just need to put break

File: src/yogita/FindIdenticalString.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+public class FindIdenticalString {
+	void findIdenticalString(String str1, String str2) {
+
+		boolean flag = false;
+		if (str1.length() == str2.length()) {
+			for (int i = 0; i < str1.length(); i++) {
+				char ch1 = str1.charAt(i);
+				char ch2 = str2.charAt(i);
+				if (ch1 != ch2)
+					flag = true;
```
Comment (harshvegada) [inline src/yogita/FindIdenticalString.java:24]: Hi ruby, 
here you should put break; once you know that char is matching then no more iterations are require

**Review by harshvegada — APPROVED** [review]

File: src/yogita/FindIdenticalElement.java (hunk 1 above)
Comment (YogitaPOct2020) [reply src/yogita/FindIdenticalElement.java:27]: Is it necessary to use break with flag everytime  ?

File: src/yogita/FindIdenticalString.java (hunk 2 above)
Comment (YogitaPOct2020) [reply src/yogita/FindIdenticalString.java:24]: Ok.. got it..

**Pull request merged**